docs/products/caching/concepts/memory-usage.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Memory Management and persistence in Aiven for Caching
+---
+
+Learn how Aiven for Caching, compatible with legacy Redis® OSS, addresses the challenges of high memory usage and high change rate. Discover how it implements robust memory management and persistence strategies.
```

Comment:
i think we should add version 7.2.4

Reply:
Added this - This service is compatible with legacy Redis® OSS up to version 7.2.4, facilitating seamless transitions and compatibility.


is that what you intended?